### ISSUE TITLE
Create integration points with the CRDC-H generation workflow

### ISSUE THREAD

**Issue by jiaola** [cancerDHC/ccdh-terminology-service#29]
- Create an action in the ccdhmodel github repo to initiate the refreshing script in the terminology service
- implement a way to call an action in the ccdhmodel github once the refreshing is done.

**Comment by jiaola** [comment]
Duplicate of #26
